mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -340,6 +340,10 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 a
 		ARG 2 b
 		ARG 4 c
+	METHOD method_34945 (Lnet/minecraft/class_243;Lnet/minecraft/class_243;Lnet/minecraft/class_238;)Z
```

Comment:
Could this method be named?

Reply:
I don’t know – it’s not used anywhere in the code. It seems to be some kind of line–box collision detection algorithm, but I can’t find any code in the wild that looks like it.